crates/blockifier/src/execution/entry_point_test.rs

```diff
@@ -548,3 +552,12 @@ Unknown location (pc=0:62)
         other_error => panic!("Unexpected error type: {other_error:?}"),
     }
 }
+
+#[test]
+#[cfg(feature = "parity-scale-codec")]
+fn test_scale_trait_derivation() {
+    let call_info = CallInfo::default();
```

Comment:
This only works coz decode of ignored fields will be set to Default too.
If you pass actual value, this test will fail. So I would rally have no `skip` in those encode